concrete_sigar_test.go

```diff
@@ -74,3 +74,18 @@ func TestConcreteFileSystemUsage(t *testing.T) {
 	fsusage, err = concreteSigar.GetFileSystemUsage("T O T A L L Y B O G U S")
 	assert.Error(t, err)
 }
+
+func TestConcreteGetFDUsage(t *testing.T) {
+	concreteSigar := &sigar.ConcreteSigar{}
+	fdUsage, err := concreteSigar.GetFDUsage()
+	// if it's not implemented, don't test
+	if _, ok := err.(*sigar.ErrNotImplemented); ok {
+		println("Skipping *ConcreteSigar.GetFDUsage test because it is no implemented for " + runtime.GOOS)
+		return
+	}
+	if assert.NoError(t, err) {
+		assert.True(t, fdUsage.Open > 0)
+		assert.True(t, fdUsage.Open <= fdUsage.Max)
+		println("Testing fdUsage")
```

Comment:
I don't think this is necessary, but you want to keep it I recommend using `t.Log()`.